ansible/roles/supervisor/tasks/main.yml

```diff
@@ -4,6 +4,7 @@
   shell: 'which supervisord'
   register: which_supervisord
   changed_when: false
+  failed_when: which_supervisord.rc != 0 and which_supervisord.rc != 1
```

Comment:
The point is that I'm trying to calculate whether supervisor is installed, so I don't want it to _fail_ if it's not installed, I want to be able to use the answer in the next tasks. (If a task fails, it doesn't move on to the next one.)